docs/configuration/downstream/pull_from_upstream.md

```diff
@@ -70,7 +70,17 @@ Use a repository you maintain as we will create issues about failures here.
 ## Optional parameters
 
 * **dist_git_branches** - a (list of) branch(es) in dist-git where packit should work (defaults to `main`).
-  You can also use the [aliases provided by Packit](/docs/configuration#aliases)
+  Or a dict whose *keys* are the names of the dist-git branches where packit synchronizes release changes 
```

Comment:
would maybe add it as 2 bullet points, so that it is clear (the list/dict thing)